configure.ac

```diff
@@ -179,6 +184,13 @@ if test "$libzmq_found" = 0; then
         AC_MSG_ERROR(Required library libzmq not found; use LDFLAGS to specify library location)
 fi
 
+if test "x$enable_oqs" = "xyes"; then
+  AC_DEFINE([ENABLE_OQS], [1], [Enable OQS support])
+  AM_CONDITIONAL([ENABLE_OQS], [test "x$enable_oqs" = "xyes"])
```

Comment:
We definitely need some `AC_CHECK_LIB()` lines here to confirm that the correct version of libssl and liboqs is installed on the system before allowing anyone to enable OQS.

See the other examples of AC_CHECK_LIB() in this file to get an idea of how to do this.